### DISCUSSION TITLE
What are the two LEDs for ?

### DISCUSSION THREAD

**Discussion by dbozec** [mercenaruss/zigbee-stick-v4#17] [Q&A] [answered]
It's said that there are two LEDs for indication, but I can't find to indicate what ?
I figured out that the green LED is used to display that the device is in pairing mode, but I didn't find what's the purpose of the red one.
Can someone tell me what's the purpose of the red LED ?

**Comment by mercenaruss** [comment, marked as answer]
I used custom firmware to activate both of them. I will drop you the link here, is following the same Koenkk firmware but using both leds.

https://github.com/jethome-ru/zigbee-firmware/blob/master/ti/coordinator/cc2652/CC1352P2_CC2652P_launchpad_coordinator_20230707.zip
